cmd/azure.go

```diff
@@ -116,3 +117,9 @@ func newVirtualNetworksClient(azureSubscriptionID string, servicePrincipalToken
 
 	return &c
 }
+func newVirtualNetworkGatewayConnectionsClient(azureSubscriptionID string, servicePrincipalToken *adal.ServicePrincipalToken) *network.VirtualNetworkGatewayConnectionsClient {
```

Comment:
I still find the naming in azure packages weird. It feels like they have a competition for the longest possible name for something. (nothing you can do about this though)

Reply:
Would `VNetGatewayConnections` help ? 10 characters shorter, not bad

Reply:
Don't worry about it, this is fine.